src/enums/index.ts

```diff
@@ -13,9 +14,8 @@ export * from './nigsMessage'
 export * from './paymentMethod'
 export * from './roles'
 export * from './routes'
-export * from './staffPaymentsOptions'
-export * from './effectOfOrderTypes'
+export * from './saveErrorReasons'
 
 // external enums
 export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
-export { ActionTypes } from '@bcrs-shared-components/enums'
+export { ActionTypes, StaffPaymentOptions } from '@bcrs-shared-components/enums'
```

Comment:
Now using Staff Payment Options enum from shared library. I deleted the local one.